app/config/FrontendAppConfig.scala

```diff
@@ -17,15 +17,7 @@
 package config
 
 import com.google.inject.{Inject, Singleton}
-import models.{
-  C79Certificate,
-  CashStatement,
-  DutyDefermentStatement,
-  FileRole,
-  PostponedVATStatement,
-  SecurityStatement,
-  UserAnswers
-}
+import models.{C79Certificate, CDSCashAccount, DutyDefermentStatement, FileRole, PostponedVATStatement, SecurityStatement, UserAnswers}
```

Comment:
Better to revert as the last style (imports on separate line) or break after PostponedVATStatement as it making a very long line

Reply:
Reverted to previous style